client/src/main/java/org/apache/cassandra/sidecar/client/SidecarClient.java

```diff
@@ -80,9 +83,9 @@ public SidecarClient(SidecarInstancesProvider instancesProvider,
     public CompletableFuture<HealthResponse> sidecarHealth()
     {
         return executor.executeRequestAsync(requestBuilder()
-                                            .sidecarHealthRequest()
-                                            .retryPolicy(new OncePerInstanceRetryPolicy())
-                                            .build());
+                .sidecarHealthRequest()
+                .retryPolicy(oncePerInstanceRetryPolicy)
+                .build());
```

Comment:
The indentation looks wrong.

Reply:
Reverted to the previous state here and in several other places.